packages/teams-js/src/public/app.ts

```diff
@@ -556,7 +557,7 @@ export namespace app {
    * Gets the Frame Context that the App is running in. See {@link FrameContexts} for the list of possible values.
    * @returns the Frame Context.
    */
-  export function getFrameContext(): FrameContexts {
+  export function getFrameContext(): FrameContexts | undefined {
```

Comment:
>| undefined [](http://example.com/codeflow?start=51&length=11)

Since `GlobalVars.frameContext` can be `undefined`, that is reflected here as well.